CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.0)
 
+option(EXTLIB_FROM_SUBMODULES "External libs from submodules" OFF)
+option(MVD_INSTALL_EXPORTS "Generate Cmake project exports" ON)
```

Comment:
Ah, sorry. Didn't grok that this was really just the export files. But why would we want to disable only those?

Reply:
Installing the exports is giving problems mentioning the include path is in the same source tree
probably requires fixing in HighFive which would take long (edited)

Reply:
But why install anything in that case?

I just test drove it with the `else` clause removed, and that seemed to work pretty well with `python setup.py build`. With a generic `MVD_INSTALL` option, we could also use MVDTool as a private submodule somewhere and disable global installation if the headers are not exposed?

Reply:
> MVDTool as a private submodule
Do you mean `add_subdirectory(`? We have the `EXCLUDE_FROM_ALL` to avoid building and installing targets.

The rationale behind installing the main target would be to still help people with third party projects, which could find HighFive themselves. We don't need to install it for its own bindings. So it's debatable, maybe not worth it. If the other option (without the else) help in some case, sure, we can change.

Reply:
Couldn't there be the reverse argument and the `else` clause only added if someone has a otherwise unsolvable need for it? I would much prefer that as IMO all C++ installations should ship with CMake glue.

Reply:
MVDtool is not that popular anymore. Not worth to spend much though. Let's just drop that